src/r/recordTypes/Project.ts

```diff
@@ -102,5 +103,6 @@ export const projectPropertyDefaults:  Record<ProjectProperty, unknown> = {
   [ProjectProperty.project_metadata_title]: "",
   [ProjectProperty.project_metadata_description]: "",
   [ProjectProperty.project_metadata_tags]: [],
-  [ProjectProperty.use_custom_branding]: false
+  [ProjectProperty.use_custom_branding]: false,
+  [ProjectProperty.initial_graphics_setting]: "adaptive",
```

Comment:
Create and export and enum with all possible options for graphics settings. Export from namespace sn, for reference look at SceneType enum. Use that everywhere

Reply:
Resolving this for now @ratankhalsa92 , @amitrajput1992 
Sending a commit with the required change